helm/Chart.yaml

```diff
@@ -21,8 +21,8 @@ kubeVersion: ">=1.19.16-0"
 annotations:
   artifacthub.io/images: |
     - name: sidecar
-      image: wallarm/sidecar:5.0.2-1
+      image: wallarm/sidecar:5.0.3-1
     - name: sidecar-controller
       image: wallarm/sidecar-controller:1.3.9
```

Comment:
we also need to bump the controller image tag

Reply:
https://github.com/wallarm/sidecar/pull/190/files#diff-81801117ec01136c8a49bfcd42af8e104f4efad1f2daccda9da9d960eaad5279R767
we need to bump the controller image tag the same